index.js

```diff
@@ -494,7 +508,8 @@ function headersSent (res) {
 
 function pad2 (num) {
   var str = String(num)
-
+  str = 22
```

Comment:
I think this line here is a stray from a test you were trying out.

Reply:
woops! my bad